docs/ApplicationUploadGuide.md

```diff
@@ -0,0 +1,52 @@
+## How to upload an application##
```

Comment:
Remove `##` at the end of the heading

Reply:
![grafik](https://user-images.githubusercontent.com/1366654/31815958-99542722-b58e-11e7-9278-6c67512f24a4.png)